onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -800,6 +811,13 @@ def to_model_proto(
                 for domain, version in unique_custom_domains.items()
             ]
         )
+        # Include the library shared opset domain
+        # TODO: Remove after https://github.com/microsoft/onnxscript/issues/834 is fixed
+        onnx_model.opset_import.append(
+            onnx.helper.make_opsetid(
```

Comment:
Could we be double appending opsetid? (appended first here, then somewhere else where it is appended normally for torchlib functions)

Reply:
nvm, I think we won't double append unless we register "common" domain ops in pytorch exporter registry.